Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.5
+
+# Setup rapstore-builder
+COPY .    /code/builder
+WORKDIR   /code/builder
+RUN python3 setup.py install
+
+# Hardwritten configuration
+ENV LOGDIR  /vol/log/
```

Comment:
There's also a trick to do this [here](https://syslog-ng.com/blog/collecting-logs-containers-using-docker-volumes/)

Reply:
From what I understood, they describe a configuration to automatically use the log files messages in syslog-ng without specific configuration in the application.
So it would still require writing logs in files in a volume. And also have syslog running.

However in that case you don't have any log level information, everything is set to `"PRIORITY":"notice"`.
Python Logging can also directly send to the syslog socket without needing to use the file trick which would give log levels too.

Reply:
yes, it's more a way to collect several logs :)